pandas/core/series.py

```diff
@@ -2952,7 +2951,7 @@ def _dir_additions(self):
 Series._add_numeric_operations()
 Series._add_series_only_operations()
 Series._add_series_or_dataframe_operations()
-_INDEX_TYPES = ndarray, Index, list, tuple
+_INDEX_TYPES = np.ndarray, Index, list, tuple
```

Comment:
u can try deleting and if test passes then ok

too bad linter doesn't find this